app/templates/preprints/view.hbs

```diff
@@ -3,74 +3,51 @@
-                        </div> {{!END TEXT COL 8}}
-                        <div class="col-md-4"> {{!BUTTON COL 4}}
-                            <a type="button" class="btn btn-primary" href={{concat model.baseUrl model.project}}>
-                                View project
-                            </a>
-                        </div> {{!END BUTTON COL 4}}
-                    </div>
-                    <br>
+                    <div class="p-md osf-box-lt" id="download_project">{{!SHARE ROW}}
+                        <i class="fa fa-download" aria-hidden="true"></i>
+                        <a href={{model.downloadLink}}>Download</a>
+                        <i class="fa fa-edit" aria-hidden="true"></i>
+                        {{#link-to "preprints.edit" model.preprint.id}}Edit{{/link-to}}
+                        <span class="pull-right">
+                            {{share-popup}}
+                        </span>
+                    </div>{{!END SHARE ROW}}
 
-                    {{!CONTENT}}
-                    {{!TODO: Make the creation of these fields dynamic and generally better}}
-                    <div class="col-md-12"> {{!COL 12}}
-                        <h3> Abstract: </h3>
-                        <p> {{model.preprint.abstract}} </p>
-                    </div> {{!END COL 12}}
-
-                    <div class="col-md-12"> {{!COL 12}}
-                        <h3> Subject: </h3>
-                        <p> {{model.preprint.subject}} </p>
-                    </div> {{!END COL 12}}
-
-                    <div class="col-md-12"> {{!COL 12}}
-                        <h3> Tags: </h3>
-                        <p> {{model.preprint.tags}} </p>
-                    </div> {{!END COL 12}}
-
-                    <div class="col-md-12"> {{!COL 12}}
-                        <h3> DOI: </h3>
-                        <p> {{model.preprint.doi}} </p>
-                    </div> {{!END COL 12}}
-
-                    <div class="col-md-12"> {{!COL}}
-                        <h3> Citation: </h3>
-                        <p>
-                            <strong>APA</strong>
-                            <br>
-                            Cummings, J. N., Butler, B., & Kraut, R. (2002). The quality of online social
-                            relationships. Communications of the ACM, 45(7), 103-108.
-                        </p>
-                    </div> {{!END COL}}
+                    <h3> Abstract: </h3>
+                    <p> {{model.preprint.abstract}} </p>
 
+                    <h3> Citation: </h3>
+                    <p>
+                        <strong>APA</strong>
+                        <br>
+                        Cummings, J. N., Butler, B., & Kraut, R. (2002). The quality of online social
+                        relationships. Communications of the ACM, 45(7), 103-108.
+                    </p>
+                    <br>
```

Comment:
Don't use <br> for layout (to create space between things). use ```<p class="m-b-md">``` or something similar